be/src/io/shared_buffered_input_stream.h

```diff
@@ -53,6 +53,10 @@ class SharedBufferedInputStream : public SeekableInputStream {
         return _stream->skip(count);
     }
 
+    StatusOr<std::unique_ptr<NumericStatistics>> get_numeric_statistics() override {
+        return _stream->get_numeric_statistics();
```

Comment:
Is this why we don't get hdfs profile now?

Reply:
yes, `SharedBufferedInputStream` didn't override `get_numeric_statistics()` function.